docz/examples/example-currency.mdx

```diff
@@ -0,0 +1,53 @@
+---
+name: Currency Example
```

Comment:
Can you change this file as column types example. We should make group the similar examples.

Reply:
Ok.